ansible/roles/run-test-target/tasks/main.yml

```diff
@@ -19,7 +19,7 @@
   # which is quite significant for a specific test suite
   # (i.e. when testing during development) but is only
   # minor when running on CI
-  when: collector_test == 'ci-integration-tests'
+  when: collector_test == 'ci-integration-tests' and vm_arch == 'amd64'
```

Comment:
Not really sure about the `vm_arch == 'amd64'` here :man_shrugging:

Reply:
It was to reduce flakes on the multi-arch VMs primarily. That was when the VM resources were a bit limited but I've bumped it now in this PR so I might be able to revert this